.github/workflows/build_and_test.yml

```diff
@@ -294,28 +294,28 @@ jobs:
         ./build/mvn $MAVEN_CLI_OPTS -DskipTests -Pyarn -Pmesos -Pkubernetes -Phive -Phive-thriftserver -Phadoop-cloud -Djava.version=11 install
         rm -rf ~/.m2/repository/org/apache/spark
 
-  scala-213:
-    name: Scala 2.13 build
+  scala-213-sbt:
+    name: Scala 2.13 build with SBT
     runs-on: ubuntu-latest
     steps:
     - name: Checkout Spark repository
       uses: actions/checkout@v2
-    - name: Cache Maven local repository
+    - name: Cache Ivy local repository
       uses: actions/cache@v2
       with:
-        path: ~/.m2/repository
-        key: scala-213-maven-${{ hashFiles('**/pom.xml') }}
+        path: ~/.ivy2/cache
+        key: scala-213-ivy-${{ hashFiles('**/pom.xml', '**/plugins.sbt') }}
         restore-keys: |
-          scala-213-maven-
+          scala-213-ivy-
     - name: Install Java 11
       uses: actions/setup-java@v1
       with:
         java-version: 11
-    - name: Build with Maven
+    - name: Build with SBT
       run: |
-        export MAVEN_OPTS="-Xmx2g -XX:ReservedCodeCacheSize=1g -Dorg.slf4j.simpleLogger.defaultLogLevel=WARN"
-        export MAVEN_CLI_OPTS="--no-transfer-progress"
+        mkdir -p ~/.ivy2/local
         mkdir -p ~/.m2
```

Comment:
I think we can remove `mkdir -p ~/.m2` here and `rm -rf ~/.m2/repository/org/apache/spark` as we don't cache `.m2` anyway here.

Reply:
`.m2` is required because `publishLocal` installs artifacts into `.m2`.
But, as I mentioned above, I think we can only do `compile` and `test:compile` so I'll remove this part.